source/dynamic_data_attachment.md

```diff
@@ -0,0 +1,97 @@
+# Dynamic Data Attachment
+
+This support article illustrates how data from one project can be linked dynamically and referenced by another project(s). It also aims to demonstrate some practical examples that this feature supports.
+
+Now that this feature is available with KoBoToolbox, users can do everything that a [pulldata function](pull_data_kobotoolbox.md) does without having to maintain a separate CSV file. The data stored in the survey project (that the user intends to link) acts as a source file. Users could keep collecting data for their source files and simultaneously use them in another survey project(s). Thus, this feature should help users create and manage a longitudinal survey project _(longitudinal data is collected through a series of repeated observations of the same subjects over a short or long period)_. The best part of this feature is that it also works with a shared project where the project admin could also restrict specific permissions while sharing.
```

Comment:
> Now that this feature is available with KoBoToolbox

I would maybe avoid saying this as it's more similar to an announcement rather than a reference. This would be better suited in a blog post discussing the new feature.

So rather have something like:

> Dynamically linking projects allows for the same powerful capabilities as the [`pulldata()` function](pull_data_kobotoolbox.md), but without the need to maintain a separate CSV file.

Reply:
> Users could keep collecting data for their source files and simultaneously use them in another survey project(s)

Rather something like

> Users can continue collecting data in the _parent project_ and simultaneously use that data in other linked _child projects_

Reply:
> The best part of this feature is that it also works with a shared project where the project admin could also restrict specific permissions while sharing.

It might be best not to bring opinion into the reference article 🙂 So rather something like:

> Users can also dynamically link a shared project where the project admin can restrict specific sharing permissions.

I'm also not 100% what "the project admin could also restrict specific permissions while sharing" means in this context — maybe just explain that a bit more.